stdplugins/sca.py

```diff
@@ -1,19 +1,29 @@
 """Send Chat Actions
-Syntax: .sca <option>
-Options: typing, contact, game, location, voice, round, video, photo, document, cancel"""
+Syntax: .sca <option> <time in sec>
+        .scaoptions"""
 
 import asyncio
 from uniborg.util import admin_cmd
 
 
-@borg.on(admin_cmd(pattern="sca ?(.*)"))
+@borg.on(admin_cmd("sca (.*) (.*)"))
```

Comment:
you need a pattern= here and in all admin_cmd decorators

Reply:
oops I forgot that